### PR TITLE
OBAM-124,125 Stock Transfer API

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

**Comment by jmiranda** [comment]
Started reviewing tonight, but I'll need to spend more time on this tomorrow.

**Comment by awalkowiak** [comment]
@jmiranda tests failed because I was writing this on top of the OrderType refactor PR. We have a release branch, can we merge this refactor now?

File: src/js/components/stock-transfer/StockTransferCheckPage.jsx [hunk 1]
```diff
@@ -119,73 +122,19 @@ class StockTransferSecondPage extends Component {
 
   fetchStockTransfer() {
     this.props.showSpinner();
-
-    const url = `/openboxes/api/stockTransfer/${this.props.match.params.id}`;
+    const url = `/openboxes/api/stockTransfers/${this.props.match.params.stockTransferId}`;
```
Comment (jmiranda) [inline src/js/components/stock-transfer/StockTransferCheckPage.jsx:125]: Do we actually need the /openboxes here?

File: src/js/components/stock-transfer/CreateStockTransfer.jsx [hunk 2]
```diff
@@ -117,71 +117,45 @@ class CreateStockTransfer extends Component {
...
 
   /**
    * Sends all changes made by user in this step of stock transfer to API and updates data.
    * @public
    */
-  // TODO after API for creating stock transfer implemented
   createStockTransfer() {
-    this.props.nextPage();
+    this.props.showSpinner();
+    const url = '/openboxes/api/stockTransfers/';
```
Comment (jmiranda) [inline src/js/components/stock-transfer/CreateStockTransfer.jsx:142]: Did we solve the context path problem in the develop branch or was that just in the Grails upgrade branch?

**Review by jmiranda — APPROVED** [review]
Besides the questions, this looks good to me.

File: src/js/components/stock-transfer/CreateStockTransfer.jsx (hunk 2 above)
Comment (awalkowiak) [reply src/js/components/stock-transfer/CreateStockTransfer.jsx:142]: It was grails upgrade branch

File: src/js/components/stock-transfer/StockTransferCheckPage.jsx (hunk 1 above)
Comment (awalkowiak) [reply src/js/components/stock-transfer/StockTransferCheckPage.jsx:125]: In 1.3.9 yes, in 3.3.10 this will be not required

**Review by pmuchowski — APPROVED** [review]

**Pull request merged**